### PR TITLE
Refactoring imports

### PR DESCRIPTION
**Pull request by anna-wro**
Before:
```
import ButtonPrimary from '../../buttons/ButtonPrimary';
import DocsBlock from '../../../docs/DocsBlock';
```
After:
```
import ButtonPrimary from 'buttons/ButtonPrimary';
import DocsBlock from 'DocsBlock';
```

Closes #1135

### CODE REVIEW

File: src/components/avatar/Avatar.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from '../icons/Icon';
+import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from 'icons/Icon';
```
Comment (aju) [inline src/components/avatar/Avatar.jsx]: @kdzwinel @vergilius @matikrk this way or from 'components/icons/Icon';
same question for 'DocsBlock' or 'docs/DocsBlock' ?

Comment (matikrk) [inline src/components/avatar/Avatar.jsx]: DocsBlock will be moved soon to components (on @kdzwinel PR about interactive pages). So icons/Icon is ok for me 👍

File: package.json [hunk 2]
```diff
@@ -60,5 +60,8 @@
     "postversion": "git push && git push --tags",
     "watch": "gulp build && gulp watch & npm start",
     "test": "npm run scss-lint -s && npm run scss-unused-variables -s  && npm run js-lint -s  && jest"
+  },
+  "jest": {
+    "moduleDirectories": ["node_modules", "src/components", "src/docs"]
```
Comment (matikrk) [inline package.json]: we don't need src/docs, DocsBlock will be moved

File: scripts/tasks/build-react-pages.js [hunk 3]
```diff
@@ -41,6 +38,14 @@ module.exports = function(gulp, plugins, consts, options) {
     const jsPath = plugins.path.join(consts.VERSIONED_DIST, 'docs', relativePath);
 
     const config = Object.assign({}, coreConfig, {
+      resolve: {
+        extensions: ['.js', '.jsx'],
+        modules: [
+          consts.COMPONENTS,
+          consts.DOCS,
```
Comment (matikrk) [inline scripts/tasks/build-react-pages.js]: we don't need src/docs, DocsBlock will be moved

Comment (anna-wro) [inline scripts/tasks/build-react-pages.js]: I know, but right now we do need this :)

**Comment by anna-wro** [comment]
@aju, @matikrk - I refactored all imports

File: package.json (hunk 2 above)
Comment (anna-wro) [inline package.json]: I'll remove it after @kdzwinel merges his changes :)

Comment (kdzwinel) [inline package.json]: 👍

File: src/components/avatar/Avatar.jsx (hunk 1 above)
Comment (kdzwinel) [inline src/components/avatar/Avatar.jsx]: 👍

Comment (anna-wro) [inline src/components/avatar/Avatar.jsx]: @matikrk I saw @kdzwinel's PR and he moved DocsBlock and ContrastBox to `docs/components/`, not 'components'

**Pull request merged**